core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -111,7 +114,7 @@ class HttpGoogleServicesDAO(
     }
   }
 
-  def allowGoogleCloudStorageWrite(bucketName: String): Unit = {
+  def allowGoogleCloudStorageWrite(bucketName: String)(implicit retryCount: Int = 0): Unit = {
```

Comment:
Yeah, I suppose it is. I made it implicit mainly to avoid editing too much code, but I suppose laziness is not a great reason for using implicits. I'm tinkering with encapsulating this type of logic in an `InstrumentedRetry` trait, so this might change.